hadoop-hdds/interface-client/src/main/proto/hdds.proto

```diff
@@ -267,7 +267,6 @@ message AddScmResponseProto {
 enum ReplicationType {
     RATIS = 1;
     STAND_ALONE = 2;
-    CHAINED = 3;
```

Comment:
I think we should keep this.

Reply:
> Thanks @DaveTeng0 for working on this.
> 
> I'm not sure why `CHAINED` replication type was added in the first place. But I don't think we should remove it as a breaking change.
> 
> If we want to hide it in CLI help, we can define a separate enum for the replication types exposed to users. See `BucketLayout` vs. `AllowedBucketLayouts` enums.
> 
> We might also consider hiding `STAND_ALONE`?

Make sense! Sure let me create a new enum for the allowed replication types! (ok will remove STAND_ALONE from the new enum as well)